lib/node_modules/@stdlib/blas/ext/base/wasm/dasumpw/docs/types/index.d.ts

```diff
@@ -0,0 +1,316 @@
+	*
+	* // Specify a vector length:
+	* var N = 3;
+	*
+	* // Define a pointer (i.e., byte offset) to the first vector element:
+	* var xptr = 0;
+	*
+	* // Write vector values to module memory:
+	* mod.write( xptr, oneTo( N, dtype ) );
+	*
+	* // Perform computation:
+	* var out = mod.ndarray( N, xptr, 1, 0 );
+	* // returns 6.0
+	*/
+	ndarray( N: number, xptr: number, strideX: number, offsetX: number ): number;
+}
+
+/**
+* Interface describing `dasumpw`.
+*/
+interface Routine extends ModuleWrapper {
+	/**
+	* Computes the sum of absolute values (L1 norm) of double-precision floating-point strided array elements using pairwise summation.
+	*
+	* @param N - number of indexed elements
+	* @param x - input array
+	* @param strideX - stride length
+	* @returns sum
+	*
+	* @example
+	* var Float64Array = require( '@stdlib/array/float64' );
+	*
+	* var x = new Float64Array( [ 1.0, -2.0, 2.0 ] );
+	*
+	* var out = dasumpw.main( 3, x, 1 );
+	* // returns 5.0
+	*/
+	main( N: number, x: Float64Array, strideX: number ): number;
+
+	/**
+	* Computes the sum of absolute values (L1 norm) of double-precision floating-point strided array elements using pairwise summation and alternative indexing semantics.
+	*
+	* @param N - number of indexed elements
+	* @param x - input array
+	* @param strideX - stride length
+	* @param offsetX - starting index
+	* @returns sum
+	*
+	* @example
+	* var Float64Array = require( '@stdlib/array/float64' );
+	*
+	* var x = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
+	*
+	* var out = dasumpw.ndarray( 4, x, 2, 1 );
+	* // returns 9.0
+	*/
+	ndarray( N: number, x: Float64Array, strideX: number, offsetX: number ): number;
+
+	/**
+	* Returns a new WebAssembly module wrapper instance which uses the provided WebAssembly memory instance as its underlying memory.
+	*
+	* @param mem - WebAssembly memory instance
+	* @returns module wrapper instance
+	*
+	* @example
+	* var Memory = require( '@stdlib/wasm/memory' );
+	* var oneTo = require( '@stdlib/array/one-to' );
+	*
+	* // Create a new memory instance with an initial size of 10 pages (640KiB) and a maximum size of 100 pages (6.4MiB):
+	* var mem = new Memory({
+	*     'initial': 10,
+	*     'maximum': 100
+	* });
+	*
+	* // Create a BLAS routine:
+	* var mod = new dasumpw.Module( mem );
+	* // returns <Module>
+	*
+	* // Initialize the routine:
+	* mod.initializeSync();
+	*
+	* // Define a vector data type:
+	* var dtype = 'float64';
+	*
+	* // Specify a vector length:
+	* var N = 3;
+	*
+	* // Define a pointer (i.e., byte offset) to the first vector element:
+	* var xptr = 0;
+	*
+	* // Write vector values to module memory:
+	* mod.write( xptr, oneTo( N, dtype ) );
+	*
+	* // Perform computation:
+	* var out = mod.main( N, xptr, 1 );
+	* // returns 6.0
+	*/
+	Module: ModuleConstructor;
+}
+
+/**
+* Adds a scalar constant to each double-precision floating-point strided array element and computes the sum using an improved Kahan–Babuška algorithm.
```

Comment:
```suggestion
* Computes the sum of absolute values (L1 norm) of double-precision floating-point strided array elements using pairwise summation.
```

Beware of copy-paste errors.